src/distortion.js

```diff
@@ -90,14 +154,15 @@ class Distortion extends Effect {
    *
    * @method set
    * @for p5.Distortion
-   * @param {Number} [amount=0.25] Unbounded distortion amount.
+   * @param {Number} [amount] Unbounded distortion amount.
    *                                Normal values range from 0-1.
-   * @param {String} [oversample='none'] 'none', '2x', or '4x'.
+   * @param {String} [oversample] 'none', '2x', or '4x'.
    */
   set(amount, oversample) {
-    if (amount) {
+    if (typeof amount === 'number') {
```

Comment:
Just for my understanding, is this to let you set just the oversample parameter by passing `.set(undefined, '2x')`?

Reply:
no, it is to solve #652, now it is possible to set the amount to 0

Reply:
Ahh got it, thanks!